src/visualization/components/Player.tsx

```diff
@@ -101,9 +108,11 @@ class Player extends React.Component<any, any> {
         let iconTooltip = this.props.playback ? "Pause playback" : "Start playback";
         let speedIconColor = this.props.speed ? "white" : "#666";
         let fullPageIconColor = this.props.fullpage ? "white" : "#666";
+        
```

Comment:
remove

Reply:
not resolved - if you disagree mark as won'tfix and give an explanation

---
In reply to: [175851011](https://github.com/Microsoft/clarity-tools/pull/41#discussion_r175851011) [](ancestors = 175851011)